kore/src/Kore/Step/Rule.hs

```diff
@@ -101,12 +106,14 @@ import Kore.Internal.TermLike
     , pattern Not_
```

Comment:
This import list is growing quite large. Let's replace it with a qualified import, `import qualified Kore.Internal.TermLike as TermLike`, so we don't have to list every item.